mbf_abstract_nav/include/mbf_abstract_nav/controller_action.h

```diff
@@ -91,6 +91,7 @@ class ControllerAction :
 
   boost::mutex goal_mtx_; ///< lock goal handle for updating it while running
   geometry_msgs::PoseStamped robot_pose_; ///< Current robot pose
+  geometry_msgs::TwistStamped robot_feedback_vel_; //Actual robot velocity feedback
```

Comment:
better `robot_vel_; ///< Actual robot velocity feedback`
(why feedback?;  and use inline docstring)

Reply:
In control field normally call sensor information be a feedback, I am worried that will not be able to clearly indicate the meaning of this robot_vel_ in the program (like cmd_vel or actual velocity). maybe follow the same naming rule call robot_actual_vel_?

Reply:
I would use `current_pose_` and `current_vel_`, as in the ExePath feedback msg, but put both attributes in the RobotInformation class (and move the odom topic there, too, as I explain in the review)

Reply:
btw, according to #191, this change won't be ever released :disappointed: